config/deploy.rb

```diff
@@ -1,10 +1,11 @@
-lock '3.4.0'
+# config valid only for Capistrano 3.1
+lock '3.2.1'
```

Comment:
@peetucket Can you address @cbeer's question (that line changed in your commit) so we can merge this? Thanks.